tests/unit/load.ts

```diff
@@ -0,0 +1,54 @@
+import assert = require('intern/chai!assert');
```

Comment:
The `assert` module is not an ES6 module, so it (more than likely) shouldn't be imported with ES6 semantics. This may have been fixed in later builds of TS (I've seen instances where `import * as ...` isn't callable), but it's still confusing since in ES6, `assert` would be a non-callable object.